src/client/sandbox/event/simulator.ts

```diff
@@ -610,7 +620,16 @@ export default class EventSimulator {
 
     _dispatchMouseEvent (el, args, { dataTransfer, timeStamp }: any) {
         const disabledParent = domUtils.findParent(el, true, node => {
-            return node.hasAttribute && nativeMethods.hasAttribute.call(node, 'disabled');
+            let disableable = false;
+
+            for (const isDisableableHTMLElementType of DISABLEABLE_HTML_ELEMENT_TYPE_CHECKERS) {
+                if (isDisableableHTMLElementType(node)) {
+                    disableable = true;
+                    break;
+                }
```

Comment:
I would prefer to extract the `isDisableableHTMLElementType` logic to the separate method. It's optional to you. If you prefer to leave as is, please add an empty line after assignment.